project/plugins.sbt

```diff
@@ -0,0 +1,12 @@
+resolvers += Resolver.typesafeRepo("releases")
+resolvers += Resolver.JCenterRepository
+
+addSbtPlugin("com.codacy" % "codacy-sbt-plugin" % "18.0.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.6.0")
+
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
+
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
```

Comment:
If you are using `codacy-sbt-plugin` you should not include coverage related plugins since they are already brought.

Reply:
Not for this version of sbt...

Reply:
Hmm, can you extend on that?